sharding-jdbc/sharding-jdbc-core/src/main/java/org/apache/shardingsphere/shardingjdbc/jdbc/core/resultset/ResultSetUtil.java

```diff
@@ -85,6 +85,8 @@ private static Object convertNullValue(final Class<?> convertType) {
     private static Object convertNumberValue(final Object value, final Class<?> convertType) {
         Number number = (Number) value;
         switch (convertType.getName()) {
+	        case "boolean":
```

Comment:
I think this change proved wrong last week by your test. What does this pr mean?

Reply:
this is another bug

Reply:
Sorry, I can't understand.
You said "Fixes #2604", but meanwhile you said "this is another bug". I can see olny one bug.

Reply:
it fixes [case 2](https://github.com/apache/incubator-shardingsphere/issues/2604#issuecomment-507971772).

Reply:
If so, the problem was not resolved completely.

Reply:
Yes, this PR should not refer to #2604, but I think Case 2 is another issue like #2654 and It can be solved first. What do you think?

Reply:
What about PostgreSQL?

>  In postgresql, 116 is true and the other numbers are false.

Reply:
PostgreSQL boolean type is case 1,  true value return a byte array([116]).

Reply:
OK. Please fix the failure of travis-ci, then I'll merge this pr.

Reply:
SCM checkout FAILURE

Reply:
Synchronize your repository from upstream and push the changes.